README.rst

```diff
@@ -8,17 +8,21 @@ SciKit-Learn Laboratory
 .. image:: https://img.shields.io/coveralls/EducationalTestingService/skll/stable.svg
     :target: https://coveralls.io/r/EducationalTestingService/skll
 
-.. image:: https://img.shields.io/pypi/dm/skll.svg
-   :target: https://warehouse.python.org/project/skll/
-   :alt: PyPI downloads
-
 .. image:: https://img.shields.io/pypi/v/skll.svg
-   :target: https://warehouse.python.org/project/skll/
+   :target: https://pypi.org/project/skll/
    :alt: Latest version on PyPI
 
 .. image:: https://img.shields.io/pypi/l/skll.svg
    :alt: License
 
+.. image:: https://img.shields.io/conda/v/desilinguist/skll.svg
+   :target: https://anaconda.org/desilinguist/skll
+   :alt: Conda package for SKLL
+
+.. image:: https://img.shields.io/pypi/pyversions/skll.svg
```

Comment:
This image only lists 2.7, 3.3, and 3.4. Don't we support up to 3.6 at this point?

Reply:
It's automatically derived from PyPI so once the new package is uploaded with the new `setup.py`, the badge should change to 2.7 and 3.6.